### DISCUSSION TITLE
"Account error: Request timed out" when switching to another account

### DISCUSSION THREAD

**Discussion by europacafe** [the-djmaze/snappymail#1781] [Q&A]
It happens quite often. My main account has several accounts added. 

The timeout happens when I switch to another account. After a timeout, if I switch to another account again, it is usually ok.

Fyi, I have Avatars 1.2 extension installed.
snappymail 2.38.0
![image](https://github.com/user-attachments/assets/8b6aef4d-00c8-433c-8307-fa5e5f1baf48)

**Comment by cm-schl** [comment]
Do the logs tell you something additional?
I never have seen this message when changing accounts so maybe we should dig a bit deeper into the log files 🙂
